### PR TITLE
OT Remote Mgmt Beta

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

OT Remote Mgmt Beta (WIP) - publish 2/23

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206470921651265

### CODE REVIEW

**Review by ajaiswals — APPROVED** [review]

File: docs/send-data/opentelemetry-collector/remote-management.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+1. On the **OpenTelemetry Collection** page, click **Add Collector**.
+1. In the **Set up Collector** step, select **Linux** as the platform.<br/><img src={useBaseUrl('img/send-data/linux-install.png')} alt="linux-install" width="600"/>
+1. Enter your **Installation Token**.
+1. Under **Tag data on Collector level**, add a new tag, “application = Apache” as seen in the screenshot below to identify these collectors as Apache running on them.
+1. For **Collector Settings**, leave them as default (unchecked).
+1. Under **Generate and run the command to install the collector**, copy the command and execute it in your system terminal where the collector needs to be installed.<br/><img src={useBaseUrl('img/send-data/linux-terminal-installation.png')} alt="linux-terminal-installation" width="600"/>
+1. Wait for the installation process to complete, then click **Next** to proceed.
+1. On the next screen, you will see a list of available Source Templates. For our use case, we will select Apache Source Template.
+
+If you choose to close this screen, you can also navigate to create a new Source template by selecting **Manage Data** > **Collection** > **Source Template**.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management.md:51]:
```suggestion
:::note
If you choose to close this screen, you can navigate back to Source template creation by selecting **Manage Data** > **Collection** > **Source Template**.
:::
```

File: docs/send-data/opentelemetry-collector/remote-management.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+### Step 1: Collector installation
+
+In this step, we'll install the collector on 50 servers and add a uniquely identifiable tag to these collectors depicting that these have Apache server running on them.
+
+1. In Sumo Logic, select **Manage Data** > **Collection** > **OpenTelemetry Collection**.
+1. On the **OpenTelemetry Collection** page, click **Add Collector**.
+1. In the **Set up Collector** step, select **Linux** as the platform.<br/><img src={useBaseUrl('img/send-data/linux-install.png')} alt="linux-install" width="600"/>
+1. Enter your **Installation Token**.
+1. Under **Tag data on Collector level**, add a new tag, “application = Apache” as seen in the screenshot below to identify these collectors as Apache running on them.
```
Comment (jpipkin1) [inline docs/send-data/opentelemetry-collector/remote-management.md:45]:
```suggestion
1. Under **Tag data on Collector level**, add a new tag, “application = Apache” as seen in the screenshot below to identify these collectors as having Apache running on them.
```

File: docs/send-data/opentelemetry-collector/remote-management.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Goal: *As a user, I have 50 Linux servers where I'm running Apache servers and would like to monitor the Apache error logs from all these servers.*
+
+### Step 1: Collector installation
+
+In this step, we'll install the collector on 50 servers and add a uniquely identifiable tag to these collectors depicting that these have Apache server running on them.
+
+1. In Sumo Logic, select **Manage Data** > **Collection** > **OpenTelemetry Collection**.
+1. On the **OpenTelemetry Collection** page, click **Add Collector**.
+1. In the **Set up Collector** step, select **Linux** as the platform.<br/><img src={useBaseUrl('img/send-data/linux-install.png')} alt="linux-install" width="600"/>
```
Comment (jpipkin1) [inline docs/send-data/opentelemetry-collector/remote-management.md:43]:
```suggestion
1. In the **Set up Collector** step, select **Linux** as the platform.<br/><img src={useBaseUrl('img/send-data/linux-install.png')} alt="linux-install" style={{border: '1px solid gray'}} width="800"/>
```

File: docs/send-data/opentelemetry-collector/remote-management.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+:::note
+If you choose to close this Source template creation screen, you can navigate back by selecting **Manage Data** > **Collection** > **Source Template**.
+:::
+
+
+### Step 2: Data configuration
+
+In this step, we'll create a data collection configuration to collect Apache error logs and link them to all the collectors that have the tag "application = Apache".
+
+1. Complete the Source Template form by providing the Name, Error log file path and click **Next**.
```
Comment (jpipkin1) [inline docs/send-data/opentelemetry-collector/remote-management.md:60]: Could you clean up the wording for the above step?

File: docs/send-data/opentelemetry-collector/remote-management.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+1. On the **Link Collectors** step, you will have the option to link the collectors using the Collector name or by adding tags to find the group of collectors. For our scenario, we will add the tag "application = Apache".<br/><img src={useBaseUrl('img/send-data/local-file-apache.png')} alt="local-file-apache" width="300"/>
+1. Click **Preview Collector(s)** to see the list of collectors that will be linked to the newly created source Template.<br/><img src={useBaseUrl('img/send-data/link-collectors.png')} alt="link-collectors" width="600"/>
```
Comment (jpipkin1) [inline docs/send-data/opentelemetry-collector/remote-management.md:62]:
```suggestion
1. Click **Preview Collector(s)** to see the list of collectors that will be linked to the newly created source Template.<br/><img src={useBaseUrl('img/send-data/link-collectors.png')} alt="link-collectors" style={{border: '1px solid gray'}} width="800"/>
```

File: docs/send-data/opentelemetry-collector/remote-management.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+1. Click **Next** to complete Source Template Creation. In the background, the system will apply the configuration to all the linked collectors and start collecting from Apache error files.
```
Comment (jpipkin1) [inline docs/send-data/opentelemetry-collector/remote-management.md:63]:
```suggestion
1. Click **Next** to complete Source Template creation. In the background, the system will apply the configuration to all the linked collectors and start collecting from Apache error files.
```

**Review by jpipkin1 — APPROVED** [review]
@kimsauce, I approved contingent on your addressing my suggestions.

File: docs/send-data/opentelemetry-collector/remote-management.md (hunk 4 above)
Comment (kimsauce) [reply docs/send-data/opentelemetry-collector/remote-management.md]: done! 

`1. Complete the Source Template form by providing the **Name** and your error log **File Path**, then click **Next**.`

**Pull request merged**